src/frontend/src/modals/IOModal/components/chatView/chat-view.tsx

```diff
@@ -17,6 +17,13 @@ import useDragAndDrop from "./chatInput/hooks/use-drag-and-drop";
 import { useFileHandler } from "./chatInput/hooks/use-file-handler";
 import ChatMessage from "./chatMessage/chat-message";
 
+const MemoizedChatMessage = memo(ChatMessage, (prevProps, nextProps) => {
+  return (
+    prevProps.chat.message === nextProps.chat.message &&
+    prevProps.chat.id === nextProps.chat.id
+  );
+});
+
```

Comment:
You'll need to verify the contentblocks as well

Reply:
Added properties and session too